cmake/onnxruntime_nblas.cmake

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_definitions(-DNUPHAR_USE_AVX2=1)
```

Comment:
What if the machine doesn't have avx2?

Reply:
@baowenlei why nblas is depending on definition? nblas as an independent library also should not use any nuphar words.

Reply:
The code would detect that at runtime and fallback to MKL

---
In reply to: [277505763](https://github.com/Microsoft/onnxruntime/pull/881#discussion_r277505763) [](ancestors = 277505763)

Reply:
Calls might fallback, but this definition is also used for including headers and declaration. 
I think that is the problem snnn said.